app/webpacker/controllers/tom_select_controller.js

```diff
@@ -10,7 +10,6 @@ export default class extends Controller {
       maxOptions: null,
       plugins: ["dropdown_input"],
       allowEmptyOption: true, // Show blank option (option with empty value)
-      closeAfterSelect: true,
```

Comment:
I think this is very useful for single selects, so we should probably keep it here. 
Another way to customise it would be to set the option in the html attribute, which will get merged with these options.  Something like `data-tom-select-options-value="{closeAfterSelect: false}"`

I think  had previously also considered subclassing this class for something like this (eg `class TomSelectMultiController extends TomSelectController`) but that might not be worth it.

Reply:
`I think this is very useful for single selects, so we should probably keep it here.`

This does not affect single select hence why I removed it, from the docs:

> After a selection is made, the dropdown will remain open if in a multi-selection control or **will close in a single-selection control**. Setting closeAfterSelect to true will force the dropdown to close after selections are made. Setting closeAfterSelect to false will keep the dropdown open after selections are made.

Reply:
Ahh, thanks for explaining!